packages/ember-data/lib/serializers/embedded-records-mixin.js

```diff
@@ -121,9 +121,9 @@ var EmbeddedRecordsMixin = Ember.Mixin.create({
    @param {String} prop the hash has been referenced by
    @return {Object} the normalized hash
   **/
-  normalize: function(typeClass, hash, prop) {
-    var normalizedHash = this._super(typeClass, hash, prop);
-    return this._extractEmbeddedRecords(this, this.store, typeClass, normalizedHash);
+  normalize: function(typeClass, hash, prop, store) {
```

Comment:
I'm not really happy about adding a `store` argument here. @wecc is there a better way to give store to `_extractEmbeddedRecords`?